frontend/src/utils/schedule-helpers.ts

```diff
@@ -189,3 +189,7 @@ export function getPositionOfYearInSchedule(
   }
   return index + 1;
 }
+
+export function isYearInPast(year: number): boolean {
+  return new Date().getFullYear() > year + 1;
```

Comment:
If you prefer to keep using the actual date, you should factor in the month of the year, since once it's September, the previous academic year should be collapsed.